### DISCUSSION TITLE
Update to CMake Tools 1.20 Release Date

### DISCUSSION THREAD

**Discussion by sinemakinci1** [microsoft/vscode-cmake-tools#4157] [Announcements]
We wanted to inform you that the release date of CMake Tools version 1.20 will be delayed due to the upcoming holiday season. We are now aiming to release 1.20 in ~~January 2025~~ February 2025. You can always view the release schedule for our extension at the Wiki below. 

https://github.com/microsoft/vscode-cmake-tools/wiki/Release-schedule

Please comment below if you have any questions or feedback for our team!

**Comment by ehis-042** [comment]
Nice project
